test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,9 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() {
+        assertTrue(Utils.isAnyNull((Object) null));
```

Comment:
You can try different kinds of input supplied to `isAnyNull`. Try to test out all the corner cases as well.